src/components/Column/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { Component } from "react";
+/**
+ * Column properties.
+ */
+export interface IColumnProps {
```

Comment:
Pra ficar menos estranho, so colocaria a definição da interface TS embaixo do componente ao inves de em cima. Se for possivel.